helm/values.yaml

```diff
@@ -50,6 +50,17 @@ manager:
     python:
       repository: ghcr.io/open-telemetry/opentelemetry-operator/autoinstrumentation-python
       tag: 0.43b0
+  autoAnnotation:
```

Comment:
`autoAnnotateAutoInstrumentation`?  I'm fine either way, just another possibility for you to think about...

Reply:
Yeah I was debating the same. Wasnt sure if thats too verbose.. but maybe that is more explicit & clear.

Reply:
It is definitely verbose - but nobody is typing it in by hand :)